src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -30,12 +31,25 @@ def asdict_without_none(obj):
     return asdict(obj, dict_factory=lambda x: {k: v for (k, v) in x if v})
 
 
+def get_input_type(input, input_type_mapping) -> ValueType:
+    if input not in input_type_mapping:
+        return ValueType.STRING
+
+    if input_type_mapping[input] == "image":
+        return ValueType.IMAGE
+    elif input_type_mapping[input] == "object":
```

Comment:
Since the PR is targeting to support "image" in prompts, maybe it is not a good idea to handle "object" here

Reply:
Will separate it from this PR.